lte/gateway/c/core/oai/tasks/amf/amf_Security_Mode.cpp

```diff
@@ -90,10 +90,11 @@ int amf_handle_security_complete_response(
       }
     }
 
-    /* FSM takes care of sending initial context setup request */
-    ue_state_handle_message_initial(COMMON_PROCEDURE_INITIATED1,
-                                    STATE_EVENT_SEC_MODE_COMPLETE, SESSION_NULL,
-                                    ue_mm_context, amf_ctx);
+    // s6a update location request
+    int rc = amf_send_n11_update_location_req(ue_mm_context->amf_ue_ngap_id);
+    if (rc == RETURNerror) {
+      OAILOG_INFO(LOG_AMF_APP, "AMF_APP: n11_update_location_req failure\n");
```

Comment:
Change the log level to ERROR and message description shall indicate that "AMF has failed to send Update Location Request ".
While logging include amf_ue_ngap_id and imsi64 for better debugging.

Reply:
addressed